src/v/cluster/scheduling/leader_balancer_constraints.h

```diff
@@ -107,12 +107,11 @@ class even_topic_distribution_constraint final
     // more than this value.
     static constexpr double error_jitter = 0.000001;
 
-    template<typename ValueType>
-    using topic_map = chunked_hash_map<topic_id_t, ValueType>;
-
 public:
     even_topic_distribution_constraint(
-      group_id_to_topic_id, const shard_index& si, const muted_index& mi);
+      const group_id_to_topic_id&,
+      const shard_index& si,
+      const muted_index& mi);
```

Comment:
should this change be a part of previous commit ?

Reply:
You mean the change to using group_to_topic by reference? Probably should be a commit on its own.